src/browser/modules/Editor/ActionButtons.tsx

```diff
@@ -33,8 +34,8 @@ interface ActionButton {
   iconColor?: string
 }
 
-const ActionButtons: React.FC<ActionButtonProps> = ({ buttons }) => (
-  <ActionButtonSection containerWidth={buttons.length * 33}>
+const ActionButtons = ({ buttons, width = 24 }: ActionButtonProps) => (
```

Comment:
Moved away from React.FC after reading this: https://github.com/facebook/create-react-app/pull/8177